lib/main.dart

```diff
@@ -25,64 +88,149 @@ Widget build(BuildContext context) {
               backgroundImage: AssetImage('assets/images/jose.ramos.jpeg'),
               radius: 80,
             ),
-            Text(
-              'Jose Ramos',
-              style: TextStyle(
-                fontSize: 40,
-                color: Colors.black87,
-              ),
-              textAlign: TextAlign.center,
-            ),
+            FutureBuilder<PersonalInfo>(
```

Comment:
You are repeating unneccessary `FutureBuilder` widgets, you can just use one and go forward getting all the data you need